.env.bckp

```diff
@@ -0,0 +1,7 @@
+AUTH_AAD_TEST_HARNESS_PASSWORD=Ch3rrytw1$t@0rG
```

Comment:
is this file needed?

Reply:
Addressed in 25eb4f5a65de0e22a10bcddbc7317ac9e2360584